CMakeLists.txt

```diff
@@ -22,16 +22,23 @@ macro(get_all_targets_recursive targets dir)
 endmacro()
 ########################################################################
 
-# What is this for?
+# Libraries to link.
+# See https://github.com/InsightSoftwareConsortium/ITK/blob/master/CMake/ITKModuleAPI.cmake
 set(CLEsperanto_LIBRARIES CLEsperanto)
 
 # Set position independent code on all targets
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 message(STATUS "CMAKE_POSITION_INDEPENDENT_CODE IS ${CMAKE_POSITION_INDEPENDENT_CODE}")
 
+# Enable/disable ITK/CLIc test suites independently.
+# Note that clci inherits the value of BUILD_TESTING when clci is declared.
+set(ITKCLEsperanto_BUILD_TESTING ${BUILD_TESTING})
+option(CLIc_BUILD_TESTING OFF)
+set(BUILD_TESTING ${CLIc_BUILD_TESTING})
+
 # Get CLIc
 set(CLIc_GIT_REPOSITORY "https://github.com/clEsperanto/CLIc_prototype.git")
-set(CLIc_GIT_TAG        "master")
+set(CLIc_GIT_TAG        "development")
```

Comment:
Let's use a specific hash as suggested by @tbirdso and make a comment noting the branch and data the commit comes from.

Reply:
I've gone ahead and set a hash at the current `development` branch head to mitigate future CI issues.